sim/warrior/arms/arms.go

```diff
@@ -129,20 +129,10 @@ func (war *ArmsWarrior) GetWarrior() *warrior.Warrior {
 func (war *ArmsWarrior) Initialize() {
 	war.Warrior.Initialize()
 	war.RegisterSpecializationEffects()
-
-	// if war.Options.UseRecklessness {
-	// 	war.RegisterRecklessnessCD()
-	// }
-
-	// if war.Options.ClassOptions.UseShatteringThrow {
-	// 	war.RegisterShatteringThrowCD()
-	// }
-
-	// war.BattleStanceAura.BuildPhase = core.CharacterBuildPhaseTalents
+	war.RegisterMortalStrike()
 }
 
-// func (war *ArmsWarrior) Reset(sim *core.Simulation) {
-// 	war.Warrior.Reset(sim)
-// 	war.BattleStanceAura.Activate(sim)
-// 	war.Stance = warrior.BattleStance
-// }
+func (war *ArmsWarrior) Reset(sim *core.Simulation) {
+	war.Warrior.Reset(sim)
+	war.BattleStanceAura.Activate(sim)
```

Comment:
Stance activation should be done from prepull actions instead of being hardcoded here.

Reply:
Oh I meant to remove these actually, thanks for pointing it out